src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/Utilities/EnvironmentHelper.cs

```diff
@@ -33,15 +34,16 @@ public sealed class ApplicationConfig
 			public uint   system_property_count;
 			public uint   number_of_assemblies_in_apk;
 			public uint   bundled_assembly_name_width;
-			public uint   number_of_assembly_blobs;
+			public uint   number_of_assembly_store_files;
+			public uint   number_of_dso_cache_entries;
 			public uint   mono_components_mask;
 			public string android_package_name;
 		};
-		const uint ApplicationConfigFieldCount = 18;
+		const uint ApplicationConfigFieldCount = 19;
 
 		static readonly object ndkInitLock = new object ();
 		static readonly char[] readElfFieldSeparator = new [] { ' ', '\t' };
-		static readonly Regex stringLabelRegex = new Regex ("^\\.L\\.env\\.str\\.[0-9]+:", RegexOptions.Compiled);
+		static readonly Regex stringLabelRegex = new Regex ("^\\.L\\.autostr\\.[0-9]+:", RegexOptions.Compiled);
```

Comment:
I have no idea what this means or implies.

Reply:
The new assembly generator now automatically handles assigning unique labels to all strings used throughout the code. It then outputs them at the end of build in a single section. Strings used to have labels like `L.env.1.str` before (an artifact from the times when only environment variables were output, thus `env`) but now we have more strings which are NOT environment variable names or values, thus the change to `autostr` - for "automatically handled strings". `autostr` is also the name of section where the strings are kept.

Reply:
TIL! Thanks!